contents/docs/advanced/browser-extension.md

```diff
@@ -1,55 +1,60 @@
 
-One of the best things about using PostHog is, all the interactions like clicks will automatically generate events in PostHog, so you don't need to do anything else to start analyzing.
+Browser extensions can block your imports. If your extension throws errors when trying to import PostHog, use a bundler like [Rollup](https://rollupjs.org/) to bundle your JavaScript, and import it in a single `<script type="module" src="dist/bundle.js"/>` tag.
 
-If you'd like to instrument your own custom events, all you need to do is:
+</CalloutBox>
 
-```js-web
-posthog.capture('custom_event_name', {})
-```
+## Using product analytics
 
-[See our browser JS library guide for](/docs/integrate/client/js) more details
+If you have [autocapture enabled](/docs/product-analytics/autocapture), PostHog will automatically capture events when the extension interface is opened, closed, and clicked.
 
-### Session replay
+If you'd like to instrument your own custom events, use the `posthog.capture()` function:
 
-Due to the new [content security policies](https://developer.chrome.com/docs/extensions/develop/migrate/improve-security) in Manifest v3 about unsafe-eval and remote code execution, importing only `array.js` isn't enough since the `array.js` code will try to load the recorder from the PostHog CDN. This will be blocked by the browser.
+```js
+posthog.capture('custom_event_name', {})
+```
 
-To fix this instead of importing the `array.js`, you can import the `array.full.js`, follow the install step 1. but instead of using the former use the latter from PostHog's [full static assets](https://us-assets.i.posthog.com/static/array.full.js), also remember to change the `script src` to `array.full.js` or follow the install step 2. and install the [npm module for posthog.js](https://www.npmjs.com/package/posthog-js).
+See our [JavaScript web SDK docs](/docs/integrate/client/js) for more details.
 
-Import the recorder to avoid the remote code execution if using the script from the step 1.
+## Using other PostHog products
 
-```js
-import "posthog-js/dist/recorder.js"
-```
+Due to the new [content security policies](https://developer.chrome.com/docs/extensions/develop/migrate/improve-security) in Manifest v3 about unsafe-eval and remote code execution, you need to import bundles directly to use other PostHog products.
 
-Import the recorder to avoid the remote code execution if using the npm module from step 2.
+To use [session replay](/docs/session-replay), [surveys](/docs/surveys), [error tracking](/docs/error-tracking), and [web analytics](/docs/web-analytics), import their bundles directly:
 
 ```js
-import rrwebRecord from "posthog-js/dist/recorder"
-```
+import "posthog-js/dist/recorder" // For session replay
+import "posthog-js/dist/surveys" // For surveys
+import "posthog-js/dist/exception-autocapture" // For error tracking
+import "posthog-js/dist/tracing-headers" // Tracking across client and server
+import "posthog-js/dist/web-vitals" // web analytics
```

Comment:
Not really Web Analytics, just the Web Vitals part of it, most of Web Analytics is enabled by the default with the main bundle already

Reply:
Done